src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    private String excavatorName = "Excavator";
```

Comment:
here and in other machines, do we need this instance variable if there are no constructors or setters? I would just print directly